_includes/i18n_dropdown_mobile.html

```diff
@@ -2,7 +2,7 @@
   <div class="sm-hide border-bottom border-blue">
     <div class="container cntnr-wide py1 px2 lg-px0 h5">
       <div class="center">
-        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p">
+        <span id="i18n-mobile-toggle" class="block text-decoration-none white fs-13p" role="button" tabindex="0">
```

Comment:
both dropdowns get rendered in the same HTML right? so should we give them different `tabindex` values even though only one will be visible at a time?